### ISSUE TITLE
You can’t skip to the main content when using a screen reader

### ISSUE THREAD

**Issue by sssoz** [oaworks/oa-works#126]
This fails [WCAG 2.1 success criterion 2.4.1: **bypass blocks**](https://www.w3.org/TR/UNDERSTANDING-WCAG20/navigation-mechanisms-skip.html).

**Comment by sssoz** [comment]
See also #112

**Comment by thinkbulecount2** [comment]
I also wonder if adding skip links would be good; also just tried to test the skipping with Tab and it "disappears" while you're on the first fold while it goes on highlighting the links below the fold — i.e. it's not scrolling with the skips.

**Comment by sssoz** [comment]
Should’ve noted this here too, but the reason why nothing seems to be highlighted (using tabs) after the top nav, is because it navigates through the links in the slide-out menu that’s brought out by the hamburger menu icon. This is probably the main reason why we need a "skip to content" link so that we can skip not only the main nav items (PRODUCTS / IMPACT / ABOUT / TEAM) but also everything behind the secondary slide-out menu. 

A "skip to content" link could be one that is placed at the top of the page and uses the utility class `.sr-only` so that it doesn’t disrupt the layout but is still available to screen readers.

**Comment by thinkbulecount2** [comment]
To add to this, I think we should also use `.sr-only-focusable` for when the `.sr-only` element is in focus through keyboard so that a user that navigates via mouse never sees it.

As per this [Stack Overflow comment](https://stackoverflow.com/questions/34175774/what-is-the-use-of-sr-only-focusable-class-in-bootstrap).

**Comment by sssoz** [comment]
Yes! I am not really using Bootstrap, just some components and classes that are useful, like sr-only. As you can see, I only add them one at a time when I realise I need them. Feel free to add it to the utilities: https://github.com/oaworks/oa-works/blob/main/src/_includes/css/utilities.css#L1

**Comment by sssoz** [comment]
Done in https://github.com/oaworks/oa-works/pull/147/commits/89a9a7df37f22b9a6ce8f4805e434c64912f3565
Updated [a11y statement](https://oa.works/policies/accessibility/) accordingly
